### PR TITLE
"California" is keep getting added fix

### PR DESCRIPTION
**Pull request by MykytaKiriushyn**
- Now there is no "California" in every violation

### CODE REVIEW

File: o-fish-ios/Views/ReportFlow/ChooseScreens/ChooseViolationsView.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ struct ChooseViolationsView: View {
         if Settings.shared.menuData.violationCodes.count == Settings.shared.menuData.violationDescriptions.count {
             for index in Settings.shared.menuData.violationCodes.indices {
                 let violation = ViolationPickerData(
-                    caption: "California",  // TODO: Remove once agreed with design
+                    caption: "",
```
Comment (akaktus) [inline o-fish-ios/Views/ReportFlow/ChooseScreens/ChooseViolationsView.swift:16]: Are we need caption at all? Investigate oportunity to remove it.

**Comment by am-MongoDB** [comment]
The name of the violation is no longer lining up correctly in the picker 
image

**Review by akaktus — CHANGES_REQUESTED** [review]

File: o-fish-ios/Views/ReportFlow/ChooseScreens/ChooseViolationsView.swift (hunk 1 above)
Comment (am-MongoDB) [reply o-fish-ios/Views/ReportFlow/ChooseScreens/ChooseViolationsView.swift:16]: Not for violations

**Comment by am-MongoDB** [comment]
If I grep `California`, it's still appearing in the code multiple times - those should be removed

File: o-fish-ios/ViewModel/Report/ViolationViewModel.swift [hunk 2]
```diff
@@ -28,9 +28,8 @@ class ViolationViewModel: ObservableObject, Identifiable {
         if self.offence.code.isEmpty && self.offence.explanation.isEmpty {
             return ""
         } else {
-            return [[NSLocalizedString(self.offence.location, comment: ""),
-                     NSLocalizedString(self.offence.code, comment: "")].joined(separator: " "),
-                    NSLocalizedString(self.offence.explanation, comment: "")].joined(separator: "\n")
+            return [[NSLocalizedString(self.offence.code, comment: "")].joined(separator: " "),
```
Comment (akaktus) [inline o-fish-ios/ViewModel/Report/ViolationViewModel.swift:31]: .joined(separator: " ") What does you join? It's not needed.

File: o-fish-ios/Views/Components/Picker/Violation/ViolationPickerDataView.swift [hunk 3]
```diff
@@ -18,7 +17,7 @@ struct ViolationPickerDataView: View {
 
     var body: some View {
         VStack(alignment: .leading, spacing: spacing) {
-            Text([NSLocalizedString(caption, comment: ""), NSLocalizedString(title, comment: "")].joined(separator: " "))
+            Text([NSLocalizedString(title, comment: "")].joined(separator: " "))
```
Comment (akaktus) [inline o-fish-ios/Views/Components/Picker/Violation/ViolationPickerDataView.swift:20]: ^^^ The same question

**Review by akaktus — CHANGES_REQUESTED** [review]

**Review by akaktus — APPROVED** [review]

**Pull request merged**